Blanball/Views/CommonLabel/CommonLabel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CommonLabel.swift
+//  Blanball
+//
+//  Created by Nik Dub on 10.03.2023.
+//
+
+import UIKit
+
+class CommonLabel: UILabel {
```

Comment:
use Extension to UILabel if it doesn't change behaviour

Reply:
Transitioned into extension for UILabel in `Changes According PR#7`